docs/userguide/format_parse/numbers/skeletons.md

```diff
@@ -0,0 +1,301 @@
+<!--
+© 2019 and later: Unicode, Inc. and others.
+License & terms of use: http://www.unicode.org/copyright.html
+-->
+
+Number Skeletons
+================
+
+Number skeletons are a locale-agnostic way to configure a NumberFormatter in
+ICU.  Number skeletons work in MessageFormat.
+
+Number skeletons consist of *space-separated tokens* that correspond to
+settings in ICU NumberFormatter.  For example, to format a currency in compact
+notation, you could use this skeleton:
+
+    compact-short currency/GBP
+
+To use a skeleton in MessageFormat, use the "number" type and prefix the
+skeleton with `::`
+
+    {0, number, ::compact-short currency/GBP}
+
+## Syntax
+
+A token consists of a *stem* and zero or more *options*.  The stem is what
+occurs before the first "/" character in a token, and the options are each of
+the subsequent "/"-delimited strings.  For example, "compact-short" and
+"currency" are stems, and "GBP" is an option.
+
+Stems might also be dynamic strings (not a fixed list); these are called
+*blueprint stems*.  For example, to format a number with 2-3 significant
+digits, you could use the following stem:
+
+    @@#
+
+A few examples of number skeletons are shown below.  The list of available
+stems and options can be found below in [Skeleton Stems and
+Options](#skeleton-stems-and-options).
+
+## Examples
+
+| Skeleton | Input | en-US Output | Comments |
+|---|---|---|---|
+| `percent` | 25 | 25% |
+| `.00` | 25 | 25.00 | Equivalent to Precision::fixedFraction(2) |
+| `percent .00` | 25 | 25.00% |
+| `scale/100` | 0.3 | 30 | Multiply by 100 before formatting |
+| `percent scale/100` | 0.3 | 30% |
+| `measure-unit/length-meter` | 5 | 5 m |
```

Comment:
Why is the output here not "5 meters"? Is there a measure-unit width default to short or narrow forms (if so, that should be made clear somewhere)?

Reply:
Yes, unit width is short by default for all unit types (currency, percent, and measure unit).  Added a note about it.